rlclientlib/logger/http_transport_client.h

```diff
@@ -5,28 +5,30 @@
 #include "sender.h"
 #include "error_callback_fn.h"
 
-#include "utility/http_authorization.h"
 #include "utility/http_client.h"
+#include "utility/apim_http_authorization.h"
+#include "utility/eventhub_http_authorization.h"
 
 #include <pplx/pplxtasks.h>
+#include <cpprest/http_headers.h>
 
 #include <memory>
 #include "data_buffer.h"
 
+using namespace web::http;
+
 namespace reinforcement_learning {
   class i_trace;
 
-  // The eventhub_client send string data in POST requests to an HTTP endpoint.
-  // It handles authorization headers specific for the Azure event hubs.
-  class eventhub_client : public i_sender {
+  template <typename T>
```

Comment:
What is `T`? Please use a more descriptive parameter name

Reply:
Using TAuthorization.